forms-flow-theme/scss/v8-scss/_dropdownButton.scss

```diff
@@ -0,0 +1,159 @@
+// V8 Custom Dropdown Styles
+.v8-custom-dropdown {
+  &.dropdown.btn-group {
+    position: relative;
+
+    // Toggle button
+    .v8-dropdown-toggle {
+      // Variant variables (primary defaults)
+      --border-normal: #b8abff80;
+      --border-hover: #b8abff;
+      --border-active: #b8abff;
+      --bg-active: #b8abff33;
+      --divider-active: #e5e5e5;
+
+      display: inline-flex;
+      align-items: center;
+      border: 0.0625rem solid var(--border-normal) !important;
+      background: #fcfcfc !important;
+      padding: 0 !important;
+      border-radius: 1.25rem !important;
+
+      // Hide Bootstrap caret
+      &::after {
+        display: none;
+      }
+
+      // Label (first span inside toggle)
+      .label-div {
+        padding: 0.6875rem 0.875rem 0.6875rem 0.8125rem;
+        border-radius: 1.25rem 0 0 1.25rem;
+      }
+
+      .dropdown-icon {
+        padding: 0.875rem 0.875rem 0.6875rem 0.8125rem;
+        border-radius: 0 1.25rem 1.25rem 0;
+      }
+
+      .dropdown-label {
+        color: #4a4a4a;
+        font-size: 1rem;
```

Comment:
good to use the mixin metioned in the theme file.

Reply:
I have approved the pr with comments since having continueos development and integrations. It can be updated in the comming prs eather.